pkgs/development/python-modules/psycopg2/default.nix

```
@@ -38,9 +39,18 @@ buildPythonPackage rec {
 
   sphinxRoot = "doc/src";
 
-  # Requires setting up a PostgreSQL database
+  # test suite breaks at some point with:
+  #   current transaction is aborted, commands ignored until end of transaction block
   doCheck = false;
 
+  checkInputs = [
```

```suggestion
  nativeCheckInputs = [
```
I believe that using `nativeCheckInputs` would be more semantically correct